examples/plotting/trefoil.py

```diff
@@ -1,3 +1,13 @@
+''' This example plots a Radiation Warning Symbol (Trefoil). It demonstrates
+rendering annular wegdes, different arrow heads and adding arc and segment
+glyphs.
+
+.. bokeh-example-metadata::
+    :apis: bokeh.plotting.Figure.annular_wedge, bokeh.plotting.Figure.arc, bokeh.plotting.Figure.segment, bokeh.models.TeeHead, bokeh.models.VeeHead
+    :refs: :ref:`userguide_annotations` > :ref:`userguide_annotations_arrows`, :ref:`userguide_styling` > :ref:`userguide_styling_adding_mathematical_notations`
+    :keywords: trefoil, symbol, teehead, veehead, arrow, arrow head, segment, arc, circle, annular wedge
```

Comment:
```suggestion
    :keywords: trefoil, teehead, veehead, arrow, arrow head, segment, arc, circle, annular wedge
```
"symbol" is really to general/vague to have much search power